packages/dashboard/src/components/Input/MarkdownEditor/Editor.tsx

```diff
@@ -711,8 +711,8 @@ export default function Editor({ value, onChange, mode, modeSwitcher }: Markdown
 											</div>
 										</div>
 										<>
-											<div className={"prose prose-sm prose-neutral space-y-4 break-words p-4"}>
-												<div className={"w-full"} style={{ width: "600px" }}>
+											<div className={"prose prose-sm prose-neutral space-y-4 break-words p-4 w-full"}>
+												<div className={"w-full lg:w-[600px]"}>
```

Comment:
Only set the fixed width for large screen so the editor content doesn't overflow in small screen

Before:

<img width="1479" alt="image" src="https://github.com/user-attachments/assets/77c9457b-cc8e-4948-957a-77efd8098088" />

After:

<img width="1482" alt="image" src="https://github.com/user-attachments/assets/dc15ba92-f9ef-441c-bddd-0fcaae9917cb" />